src/stripe-payment.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * `stripe-payment`
+ * `Payment broker for stripe`
+ *
+ * @demo demo/index.html
+ * @customElement `stripe-payment`
+ */
+import { LitElement, html, css } from "lit-element/lit-element.js";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+class StripePayment extends LitElement {
+  /**
+   * Convention
+   */
+  static get tag() {
+    return "stripe-payment";
+  }
+  /**
+   * LitElement render styles
+   */
+  static get styles() {
+    return [
+      css`
+        [hidden] {
+          display: none !important;
+        }
+        :host {
+          font-family: "Nunito Sans", -apple-system, ".SFNSText-Regular",
+            "San Francisco", BlinkMacSystemFont, "Segoe UI", "Helvetica Neue",
+            Helvetica, Arial, sans-serif;
+          align-items: center;
+          display: grid;
+          grid-gap: 12px;
+          grid-template-areas:
+            "support support"
+            "stripe submit"
+            "output output";
+        }
+        stripe-payment-request {
+          grid-area: submit/stripe/stripe/submit;
+        }
+        stripe-elements {
+          grid-area: stripe;
+        }
+        mwc-button {
+          grid-area: submit;
+        }
+        json-viewer {
+          grid-area: output;
+        }
+
+        stripe-elements:not(:defined),
+        json-viewer:not(:defined),
+        mwc-button:not(:defined) {
+          display: none;
+        }
+      `
+    ];
+  }
+  /**
+   * LitElement / popular convention
+   */
+  static get properties() {
+    return {
+      publishableKey: { type: String, attribute: "publishable-key" },
+      shippingOptions: { type: Array },
+      displayItems: { type: Array },
+      amount: { type: Number },
+      label: { type: String },
+      country: { type: String },
+      currency: { type: String },
+      paymentMethod: { type: Object },
+      error: { type: Object },
+      output: { type: Object },
+      unsupported: { type: Boolean },
+      submitDisabled: { type: Boolean },
+      paymentText: { type: String, attribute: "payment-text" },
+      debug: { type: Boolean }
+    };
+  }
+  /**
+   * HTMLElement
+   */
+  constructor() {
+    super();
+    this.submitDisabled = true;
+    this.debug = false;
+    this.displayItems = [];
+    this.shippingOptions = [];
+    this.amount = 0;
+    this.paymentText = "Submit";
+    this.label = "Purchase";
+    this.country = "US";
+    this.currency = "usd";
+    import("@power-elements/stripe-elements/stripe-payment-request.js");
```

Comment:
can import this one statically

Reply:
Can, I always default to dynamic because the chain is broken up in blocking render trees